### PR TITLE
Update dataElectionResult.json

### PR DESCRIPTION
**Pull request by Adalsteinnjons**
This protocol needs an update according to the e-voting specifications on google drive.
This is needed for the verification of the election result on the front-end.

### CODE REVIEW

**Review by CaiMusso — DISMISSED** [review]
Good fix

File: protocol/query/method/message/data/dataElectionResult.json [hunk 1]
```diff
@@ -19,10 +19,10 @@
 						"$comment": "ID of the question"
 					},
 					"result": {
-						"description": "[Array[String]] results",
+						"description": "[Array[object]] results",
 						"type": "array",
 						"items": {
-							"type": "string",
+							"type": "object",
```
Comment (pierluca) [inline protocol/query/method/message/data/dataElectionResult.json:25]: you might want to clarify the format of that object (i.e. which properties, etc.)

Comment (Adalsteinnjons) [reply protocol/query/method/message/data/dataElectionResult.json:24]: Done

**Comment by gnarula** [comment]
@pierluca I think a test in be1-go/network is flakey but that can be taken up as a separate PR

**Review by gnarula — APPROVED** [review]

**Pull request merged**